src/ControllerMethod.php

```diff
@@ -84,7 +84,7 @@ static function parse(string $context, array $definitions, ClassMethod $method,
 							} else {
 								$responseDescriptions[$statusCode] = $docNode->value->description;
 							}
-							$responses[] = new ControllerMethodResponse($docNode->value->type, $statusCode, "text/plain", new OpenApiType(type: "string"), null);
+							$responses[] = new ControllerMethodResponse($docNode->value->type, $statusCode, "application/json", new OpenApiType(type: "array", maxLength: 0), null);
```

Comment:
This is wrong. Throwing OCS*Exception definitely returns text/plain with the error message as a string.

Reply:
> This is wrong. Throwing OCS*Exception definitely returns text/plain with the error message as a string.

I added this in a talk controller:
```php
throw new OCSNotFoundException('hi');
```
And the result is:

![grafik](https://github.com/nextcloud/openapi-extractor/assets/213943/c9dcc838-e361-46df-bea1-2769f6167092)

Reply:
Uhhh, then it's maybe different handling between Controller and OCSController?

Reply:
Throwing `OCS*Exception` on NON-ocs controller should be illegal and just ignored and treaten as a normal exception....

I will test it and check.
Any app you have in mind?

Reply:
Extracted to https://github.com/nextcloud/openapi-extractor/pull/43